youcan-shop/youcan-themes
# TH-32:  Aura > Default values in theme editor > should be localized not hard coded in Arabic

## JIRA Ticket

Ticket: [TH-32](https://youcanshop.atlassian.net/browse/TH-32).

## Prerequisites


## QA Steps

* [ ] Upload Aura theme in your Prod account
* [ ] In theme-editor remove all the default sections available
* [ ] Now check the default translation based on your store language for this list:
       - Notice bar > content
       - Footer > Copyright text
       - Slider > slider image > button text
       - Featured products > title
       - Featured collection > title
       - Product slider > title & message
       - Single product > blocks > (stock, express checkout, add to cart, countdown timer)
       - Product column > button text
       - Product page > Related product > title
       - Product reviews > title
       - Thank you page > (title, subtitle, description)

## Note
- I have been forced to update `themes/aura/templates/thankyou.json` to make the dynamic default content available, another reason we cannot delete the thankyou page section from the theme editor. After testing putting custom text and then releasing it in test env it seems all good the custom text still appears and hasn't been deleted.
- There is a bug to be fixed in rich-text component of YouCan UI, sometimes you will notice that the default translation isn't available because if the component has empty value it will push by default (`<p><br></p>`) so the default translation will not be displayed
- I have also do: corrected typo mistakes, fixed text alignment, removed unused translation keys, and corrected some liquid conditions.


[TH-32]: https://youcanshop.atlassian.net/browse/TH-32?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
